# Request
string object_name
---
# Response
geometry_msgs/PointStamped object_position
